sources/Application/Player/Player.cpp

```diff
@@ -493,7 +503,9 @@ void Player::Update(Observable &o, I_ObservableData *d) {
         };
         retrigAllImmediate_ = false;
       }
-      moveToNextStep();
+      // Don't advance in audition mode
+      if (viewData_->playMode_ != PM_AUDITION)
```

Comment:
I like to use braces even for single line if statements as I think its a bit safer when future edits happen and that seems to be the convention in the existing code, but not fussed if you prefer not too.

Reply:
There are actually a lot of places on that function where no braces are used! it's a fair point tho.